DOC_CHANGES.md

```diff
@@ -6,27 +6,61 @@ Example Doc Change:
 Description of the required change.
 -->
 
-### Resources must now use `resource_name` to declare recipe DSL
+### Resources now *all* get automatic DSL
 
-Resources declared in `Chef::Resource` namespace will no longer get recipe DSL
-automatically.  Instead, `resource_name` is required in order to have DSL:
+When you declare a resource (no matter where) you now get automatic DSL for it, based on your class name:
```

Comment:
Wasn't this already true?  It might be more useful to list the cases where the behavior is changing.